### PR TITLE
code cov testing

### PR DESCRIPTION
**Pull request by jsrobertson**

### CODE REVIEW

File: backend/controllers/bulkAppointmentsConfirm.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ const bulkAppointmentsConfirmFactory = (elite2Api, logError) => {
     } catch (error) {
       return renderError(req, res, error)
     }
-
+    console.log('Code cov test')
```
Comment (JamieDraperUK) [inline backend/controllers/bulkAppointmentsConfirm.js]: Come on mate, you know better than to commit console.logs

Comment (petergphillips) [reply backend/controllers/bulkAppointmentsConfirm.js]: :astonished:

Comment (JamieDraperUK) [reply backend/controllers/bulkAppointmentsConfirm.js]: Have you considered trying to simplify it? `console.log(console.log==window.console.log) `

Comment (jsrobertson) [reply backend/controllers/bulkAppointmentsConfirm.js]: #bantz

**Pull request closed without merging**